### PR TITLE
WX-772 Update Scala to 2.13.9

### PR DESCRIPTION
**Pull request by jgainerdewar**

### CODE REVIEW

**Comment by jgainerdewar** [comment]
Marking ready for review in the hope that the currently-running tests will pass.

**Review by aednichols — COMMENTED** [review]
Are the WDL parser regenerations incidental or required for compatibility with the AST-related `.scala` changes?

**Comment by jgainerdewar** [comment]
@aednichols All the removals of the `new Integer(n)` calls are changes I made "while I was in there" to silence warnings. What with the other errors, I lost track of my intention to check in on whether it was reasonable to change that stuff. Thoughts?

File: backend/src/main/scala/cromwell/backend/standard/callcaching/RootWorkflowFileHashCacheActor.scala [hunk 1]
```diff
@@ -48,7 +48,7 @@ class RootWorkflowFileHashCacheActor private[callcaching](override val ioActor:
           // hash to become available.
           cache.put(key, FileHashValueRequested(requesters = requester :: requesters))
         case FileHashSuccess(value) =>
-          sender() ! Tuple2(hashCommand.fileHashContext, IoSuccess(requester.ioCommand, value))
+          sender() ! Tuple2[Any, Any](hashCommand.fileHashContext, IoSuccess(requester.ioCommand, value))
```
Comment (jgainerdewar) [inline backend/src/main/scala/cromwell/backend/standard/callcaching/RootWorkflowFileHashCacheActor.scala:51]: There was some change in 2.13.9 that gave us a lot of new warnings about constructions that resolve to `Any` or `Object`, which can be caused by programmer error.  considered finding some way to disable this check, but I think the changes that resolve it are actually neutral or improvements to the clarity of the code, and I ran out of places to fix before I ran out of energy to fix them.

File: common/src/main/scala/common/util/TerminalUtil.scala [hunk 2]
```diff
@@ -6,10 +6,10 @@ object TerminalUtil {
     def maxWidth(lengths: Seq[Seq[Int]], column: Int) = lengths.map { length => length(column) }.max
     val widths = (rows :+ header).map { row => row.map { s => s.length } }
     val maxWidths = widths.head.indices.map { column => maxWidth(widths, column) }
-    val tableHeader = header.indices.map { i => header(i).padTo(maxWidths(i), " ").mkString("") }.mkString("|")
+    val tableHeader = header.indices.map { i => header(i).padTo(maxWidths(i), ' ').mkString("") }.mkString("|")
     val tableDivider = header.indices.map { i => "-" * maxWidths(i) }.mkString("|")
     val tableRows = rows.map { row =>
-      val mdRow = row.indices.map { i => row(i).padTo(maxWidths(i), " ").mkString("") }.mkString("|")
+      val mdRow = row.indices.map { i => row(i).padTo(maxWidths(i), ' ').mkString("") }.mkString("|")
```
Comment (jgainerdewar) [inline common/src/main/scala/common/util/TerminalUtil.scala:12]: Using a String rather than a Char here no longer worked.

**Comment by aednichols** [comment]
@jgainerdewar those Java files are auto-generated, so we would need to update the parser-generator in order to maintainably address the warnings.

https://github.com/broadinstitute/cromwell/pull/6928/files#diff-510fbe957396d6adb63ebbb50e9547e486d6e48b1ae9ede45bd59e4c65597390L2-L14

**Review by aednichols — APPROVED** [review]
Awesome, thank you for your persistence on this 👍

**Review by kraefrei — APPROVED** [review]
Nice work! LGTM

**Pull request merged**